android/app/src/main/java/app/organicmaps/search/SearchResult.java

```diff
@@ -141,4 +142,37 @@ public Spannable getFormattedTitle(@NonNull Context context)
 
     return builder;
   }
+
+  public String getAddress(@NonNull Context context)
+  {
+    return description.region;
+  }
+
+  public Spannable getFormattedAddress(@NonNull Context context)
+  {
+      final String address = getAddress(context);
+      final String addr = address.toLowerCase();
+      String title = getTitle(context);
+      final SpannableStringBuilder builder = new SpannableStringBuilder(address);
+
+      if (highlightRanges != null)
+      {
+        final int size = highlightRanges.length / 2;
+        int index = 0;
+
+        for (int i = 0; i < size; i++)
+        {
+          final int start = highlightRanges[index++];
+          final int len = highlightRanges[index++];
+
+          final String toFind = title.substring(start,start+len).toLowerCase();
+          int start2 = addr.indexOf(toFind);
+          if(start2<0 || start2+len>=address.length()) continue;
```

Comment:
Formatting

Reply:
Could you please guide me how should i format it?

Reply:
Check how other code is formatted:
```suggestion
          if (start2 < 0 || start2 + len >= address.length())
            continue;
```

Reply:
Thankyou, will follow proper formatting next time.